app/src/services/projects.ts

```diff
@@ -236,12 +236,19 @@ const getProjectFilterQuery = (type: ProjectListingTypeFilter): string => {
     }
 }
 
-export const getProjects = async (owner?: string, first = 20, skip = 0, type?: ProjectListingTypeFilter): Promise<ProjectsResult> => {
+export const getProjects = async (
+    owner?: string | null,
```

Comment:
I like the previous signature much better. With `null` support I understand that the function can give me projects related to the `null` project (or \*no\* project) which I doubt we even have.

`undefined` is more like "unrelated" or "unspecified", or more like "I don't care".